script/parseTs.ts

```
@@ -0,0 +1,152 @@
+        if (!sourceFile.fileName.includes("Native.d.ts")){
+            continue;
+        }
+        if (sourceFile.isDeclarationFile) {
+            ts.forEachChild(sourceFile, visit);
+        }
+    }
+
+    function visit(node: ts.Node) {
+        switch (node.kind) {
+            case ts.SyntaxKind.FunctionDeclaration:
+                const functionDeclaration = node as ts.FunctionDeclaration;                
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)